graceful_controller_ros/src/graceful_controller_ros.cpp

```diff
@@ -425,7 +432,7 @@ class GracefulControllerROS : public nav_core::BaseLocalPlanner
         }
 
         // Generate next pose
-        double dt = resolution_ / vel_x;
+        double dt = (vel_x > 0.0) ?  resolution_ / vel_x : 0.1;
```

Comment:
this assumes that our costmap resolution will always correspond to our distance traveled at said velocity... doesn't it make more sense to get our delta_time (dt) from something like the hz of the controller?

Reply:
The idea here is that when *simulating* we want to collision check every costmap cell we will drive through - so the time delta is based on how quickly we roll through cells. When our vel_x is 0 we can use a fixed timestep since we don't change which costmap cell we are in.